x-pack/plugins/security_solution/public/resolver/view/panels/use_formatted_date.test.tsx

```diff
@@ -45,7 +45,7 @@ describe('useFormattedDate', () => {
       const { findByTestId } = reactRenderResult(undefined);
       element = await findByTestId(testID);
 
-      expect(element).toBeEmpty();
+      expect(element).toBeEmptyDOMElement();
```

Comment:
`toBeEmpty` is deprecated and was showing a warning in the CLI

Reply:
Thanks, created a PR for this too. I'll close mine